### ISSUE TITLE
The descriptions look bad on dark themes

### ISSUE THREAD

**Issue by Tommalla** [Soszu/TheChroniclesOfAndaria#5]
Moim zdaniem trzeba wprowadzić wewnątrzgrowe stylowanie komponentów wyświetlających tekst, bo potem wychodzi jak na screenie.
![dark theme](https://f.cloud.github.com/assets/4281640/1460819/cf90c404-4476-11e3-8af9-6bf6de53b653.png)

**Comment by Soszu** [comment]
Yes, it's a problem indeed. This is listed in [TICKETS](https://github.com/Soszu/ZjednoczenieAndarii/wiki/TICKETS). As someone told me, there should be a way to use theme-dependent style. I mean that there should be an option to tell only that text should be emphasized or highlighted, but without explicite description.
